docs/faq.md

```diff
@@ -104,3 +104,13 @@ Yes. EarlGrey supports **Test Navigator** out-of-the-box.
 
 Yes. You can set a breakpoint on any interaction. The breakpoint will be hit before that
 interaction is executed, but after all prior interactions have been executed.
+
+**Where do I find the xctest bundle?**
```

Comment:
LGTM from my end.

nit: If you don't mind could you please remove "7" from the "How does EarlGrey compare to Xcode 7’s UI Testing" FAQ above? I think it's pretty clear without the 7.